pkg/virt-controller/watch/network_test.go

```diff
@@ -270,3 +346,9 @@ func withInterfaceStatus(ifaceStatus v1.VirtualMachineInstanceNetworkInterface)
 		)
 	}
 }
+
+func vmiWithDefaultNet() *v1.VirtualMachineInstance {
```

Comment:
I do not think this is needed, the whole point of `libvmi` was to allow a simple creation of VMI objects without such helpers. It is an one-liner after all. 

Also, for completion, I would also add the interface.

Reply:
done.